crates/consensus/common/src/validation.rs

```diff
@@ -32,9 +35,16 @@ pub fn validate_header_standalone(
         return Err(ConsensusError::BaseFeeMissing)
     }
 
+    #[cfg(feature = "optimism")]
+    let wd_root_missing = header.withdrawals_root.is_none() && !chain_spec.optimism;
+
+    #[cfg(not(feature = "optimism"))]
+    let wd_root_missing = header.withdrawals_root.is_none();
+
     // EIP-4895: Beacon chain push withdrawals as operations
     if chain_spec.fork(Hardfork::Shanghai).active_at_timestamp(header.timestamp) &&
-        header.withdrawals_root.is_none()
+        header.withdrawals_root.is_none() &&
```

Comment:
Delete this line

Reply:
Which line?

Reply:
Ah wow, complete brain fart - fixing right now. Thanks!